activerecord/test/cases/explain_subscriber_test.rb

```diff
@@ -40,7 +40,7 @@ def test_collects_pairs_of_queries_and_binds
       assert_equal binds, queries[0][1]
     end
 
-    def test_collects_nothing_if_the_statement_is_not_whitelisted
+    def test_collects_nothing_if_the_statement_is_not_permitted
```

Comment:
`.._is_not_explainable`. I think "the statement is not permitted" would mean something rather different.